tests/test_gencast_with_thermalizer.py

```diff
@@ -0,0 +1,187 @@
+import torch
+
+from graph_weather.models.forecast import GraphWeatherForecaster
+
+
+def test_gencast_thermal_integration():
+    # Test with a larger grid to properly test UNet capabilities
+    # Create a 3x3 grid (9 nodes)
+    lat_lons = [(i // 3, i % 3) for i in range(9)]  # 3x3 grid
+
+    print(f"Number of lat_lons: {len(lat_lons)}")
```

Comment:
Remove print statements in tests please, if its to check anything, it should be an assert, if not, it just clutters up the outputs from pytest.